clients/privacy-center/components/Card.tsx

```diff
@@ -8,6 +11,40 @@ type CardProps = {
   onClick: () => void;
 };
 
+/**
+ * IconProviders maps to the react-icons libraries supported by Fides.
+ * The values of the enum elements are the prefix to indicate that the iconPath should use an icon library instead of a relative path or URL.
+ */
+enum IconProviders {
+  MATERIALDESIGN = "md",
+  FONTAWESOME = "fa",
+}
+
+function iconResolver(iconPath: string, description: string): JSX.Element {
+  const iconBoxSize = "32px";
+  const prefixCandidate = iconPath.toLowerCase().split(":")[0];
+  const hasPrefix =
+    Object.values<string>(IconProviders).includes(prefixCandidate);
+  if (hasPrefix) {
+    let iconType: IconType;
+    switch (prefixCandidate) {
+      case IconProviders.MATERIALDESIGN:
+        iconType = mdIcons[iconPath.split(":")[1] as keyof typeof mdIcons];
+        break;
+      case IconProviders.FONTAWESOME:
+        iconType = faIcons[iconPath.split(":")[1] as keyof typeof faIcons];
+        break;
+      default:
+        iconType = faIcons.FaRegQuestionCircle;
+        break;
+    }
+    return (
+      <Icon boxSize={iconBoxSize} as={iconType} color="complimentary.500" />
```

Comment:
I wonder if something like this could help with the case above, where the lib doesn't have the icon specified?

```suggestion
      <Icon boxSize={iconBoxSize} as={iconType ?? faIcons.FaRegQuestionCircle} color="complimentary.500" />
```